File: official/guides/getting-started/golang/create_shipment_response.go
// Iterating through rates object printing out
// the carrier, service level, rate, and id
// of each possible rate.

rates := shipment.Rates
    for i := 0; i < len(rates); i++ {
        fmt.Println(rates[i].Carrier)
        fmt.Println(rates[i].Service)
        fmt.Println(rates[i].Rate)
        fmt.Println(rates[i].ID)
    }

File: official/guides/getting-started/php/create_shipment_response.php
## Iterating through rates object printing out
## the carrier, service level, rate, and id
## of each possible rate.

foreach ($shipment->rates as $rate) {
  print_r($rate->carrier);
  print_r($rate->service);
  print_r($rate->rate);
  print_r($rate->id);
}
